advanced-api/dynamic-remotes/app1/src/App.js

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+
+function loadComponent(scope, module) {
+  return async () => {
+    // Initializes the share scope. This fills it with known provided modules from this build and all remotes
+    await __webpack_init_sharing__("default");
+    const container = window[scope]; // or get the container somewhere else
+    // Initialize the container, it may provide shared modules
+    await container.init(__webpack_share_scopes__.default);
```

Comment:
Where can I read about this?

Reply:
source code, or the webpack PR that was opened